src/client.rs

```diff
@@ -29,6 +29,7 @@ impl Crux {
     }
 
     #[cfg(not(test))]
+    #[cfg(not(feature = "mock"))]
```

Comment:
I know another way of doing this that requires just one line, since both lines are for the same objective, it may make sense to use it:

```rust
#[cfg(all(not(test), not(feature = "mock"))]
```

Maybe the way it is is more readable, gimme your opinion :slightly_smiling_face:

Reply:
We could use a `not(any())` too.
Here's the reference I've used: https://doc.rust-lang.org/reference/conditional-compilation.html

Reply:
both solutions don't work

Reply:
d01c4ae

Reply:
used all, more explicit